ocs_ci/helpers/sanity_helpers.py

```diff
@@ -330,3 +332,47 @@ def health_check_ms(
 
         finally:
             config.switch_ctx(orig_index)
+
+    def retry_create_resources_on_ms_consumers(self, tries=3, delay=60):
+        """
+        Retry creates resources for MS consumers multiple times until it succeeds.
+        In every iteration, if it fails to create the resources, it cleans up the current resources
+        before continuing to the next iteration.
+
+        Args:
+            tries (int): The number of tries to create the resources on MS consumers
+            delay (int): The delay in seconds between retries
+
+        Raises:
+            FileNotFoundError, ocs_ci.ocs.exceptions.CommandFailed: In case of file not found,
+                or in case of a command failed after the number of the given tries
```

Comment:
This doc string doesn't fit for an error like FileNotFoundError

Reply:
Okay, I will think about how to change it.

Reply:
I removed it. In the retry function, I still have this exception to catch cause I saw that there is a slight chance to occur, and I don't want that the operation failed because of this error.